### PR TITLE
Add link to BasicAuthAuthenticationPolicy manual

### PR DESCRIPTION
**Pull request by vog**
Related to https://github.com/Pylons/pyramid/issues/3066

See also https://github.com/Pylons/pyramid/issues/3079

### CODE REVIEW

File: docs/auth/basic.rst [hunk 1]
```diff
@@ -2,9 +2,13 @@ HTTP Basic Authentication Policy
...
 To adopt basic HTTP authentication, you can use build-in authentication policy
-``BasicAuthAuthenticationPolicy``
+``BasicAuthAuthenticationPolicy``.
 
-Use it something like::
+Please check the
+`BasicAuthAuthenticationPolicy <http://docs.pylonsproject.org/projects/pyramid/en/latest/api/authentication.html#pyramid.authentication.BasicAuthAuthenticationPolicy>`_
+manual entry for more details.
+
+Use it like this::
```
Comment (stevepiercy) [inline docs/auth/basic.rst]: Let's condense the above and correct the grammar of the original sentence in one swoop. As an aside, you do not need to hard wrap narrative docs.

```
To adopt basic HTTP authentication, you can use Pyramid's built-in authentication policy, `BasicAuthAuthenticationPolicy <http://docs.pylonsproject.org/projects/pyramid/en/latest/api/authentication.html#pyramid.authentication.BasicAuthAuthenticationPolicy>`_.
```

**Comment by mmerickel** [comment]
The documents have an intersphinx link to pyramid so you can use an actual reference to the object here instead of the full URL. For example 

```
:class:`pyramid.authentication.BasicAuthAuthenticationPolicy`
```

**Comment by vog** [comment]
Thanks for the feedback. I improved my pull request.

**Comment by mmerickel** [comment]
This is great. It would also be good to add the forbidden view implementation to this example I think. Or even the entirety of my gist https://gist.github.com/mmerickel/e530f16dae61cf2960d301c7c01309b0.

It can be done in a separate PR though.

**Comment by vog** [comment]
See #187.

**Pull request merged**